Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.controller.ts

```diff
@@ -307,6 +328,52 @@
                     }
                 }
 
+                function bindCriticalities() {
+                    
+                    const optionMap = criticalityOptions.reduce((acc, next, _) => {
+                        acc[next.Id] = {
+                            text: next.Name,
+                            id: next.Id,
+                            optionalObjectContext: {
+                                id: next.Id,
+                                name: next.Name,
+                                description: next.Description
+                            }
+                        };
+                        return acc;
+                    }, {});
+
+                    //If selected state is expired, add it for presentation reasons
+                    let existingChoice = null;
+                    if ($scope.contract.criticalityTypeId !== undefined && $scope.contract.criticalityTypeId !== null) {
+                        existingChoice = {
+                            id: $scope.contract.criticalityTypeId,
+                            name: `${$scope.contract.criticalityTypeName} (udgået)`
+                        };
+
+                        if (!optionMap[existingChoice.id]) {
+                            optionMap[existingChoice.id] = {
+                                text: existingChoice.name,
+                                id: existingChoice.id,
+                                disabled: true,
+                                optionalObjectContext: existingChoice
+                            }
+                        }
+                    }
+
+                    const options = criticalityOptions.map(option => optionMap[option.Id]);
+
+                    $scope.criticality = {
+                        selectedElement: existingChoice && optionMap[existingChoice.id],
+                        select2Config: select2LoadingService.select2LocalDataNoSearch(() => options, true),
+                        elementSelected: (newElement) => {
+                            var payload = { criticalityTypeId: newElement ? newElement.id : null };
```

Comment:
still does not work when the option is cleared (using the `x`)

Reply:
![image](https://user-images.githubusercontent.com/13116889/176598540-60f2af03-49f2-4328-aae0-e1be0dea11c6.png)

Reply:
Hint: Works fine in DPR - Overførselsgrundlag